canonical-websites/www.ubuntu.com
# Fixed auto-prefixing

## Done

Copied package.json commands from snapcraft.io

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that external link icons look correct (shouldn't be black box)


## Issue / Card

Fixes Issue #660.
